src/views/LatestMovieView.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div>
+
+  </div>
```

Comment:
where is content of the page. Add content or delete the page

Reply:
Deleted